src/Neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -189,6 +196,27 @@ protected internal StackItem GetScriptContainer()
             return interop.ToStackItem(ReferenceCounter);
         }
 
+        /// <summary>
+        /// The implementation of System.Runtime.LoadScript.
+        /// Loads a script at rumtime.
+        /// </summary>
+        protected internal void RuntimeLoadScript(byte[] script, CallFlags callFlags, Array args)
```

Comment:
Without return value?

Reply:
The method itself doesn't need a return value. The loaded script will push the return value onto the stack.